### PR TITLE
chore: update to angular 15

### PR DESCRIPTION
**Pull request by chimurai**
- chore: update to angular 15
- fixed broken test setup

### CODE REVIEW

File: projects/ngx-cookie/src/lib/utils.ts [hunk 1]
```diff
@@ -21,7 +21,8 @@ export function isEmpty(value: unknown): value is undefined | null | '' | EmptyA
   if (isNil(value)) {
     return true;
   }
-  if (value === {}) {
+  // FIXME: This condition will always return 'false' since JavaScript compares objects by reference, not value.ts(2839)
+  if (value === {} as unknown) {
```
Comment (chimurai) [inline projects/ngx-cookie/src/lib/utils.ts:25]: Probably a bug in existing implementation?

**Comment by chimurai** [comment]
👋 Hi @salemdar  

Friendly ping to review this PR.
Please let me know if any adjustments are needed.

**Review by salemdar — APPROVED** [review]

**Comment by salemdar** [comment]
@chimurai I was away for a while. Thank you for the PR.

**Comment by salemdar** [comment]
@chimurai would you be interested in being a maintainer?

**Pull request merged**